src/app/home/home.component.css

```diff
@@ -21,12 +21,15 @@ button {
 
 .container {
   padding: 40px;
-  width: 800px;
+  width: 950px;
   flex-shrink: 0;
   border-radius: 20px;
   border: 1px solid rgba(204, 204, 204, 0.80);
   background: #FFF;
   color: #CCCCCCCC;
+  margin-top: 20px;
```

Comment:
you can use the margin shortcut for this: 

If the margin property has four values:
```

margin: 25px 50px 75px 100px;
top margin is 25px
right margin is 50px
bottom margin is 75px
left margin is 100px
```